more/pixels/pixelConfig.js

```diff
@@ -0,0 +1,183 @@
+import harden from '@agoric/harden';
+
+import { makePixelMintKeeper } from './pixelMintKeeper';
+import { makePixelListAssayMaker } from './pixelAssays';
+
+/**
+ * `makePixelConfigMaker` exists in order to pass in more parameters
+// than makePixelConfig allows.
+ * @param  {function} makeUseObj creates a "use object", which has all
+ * of the non-ERTP methods for assets that are designed to be used.
+ * For instance, a pixel can be colored. The use object is associated
+ * with an underlying asset that provides the authority to use it. 
+ * @param  {number} canvasSize=10 the size of the gallery in pixel
+ * squares across and down
+ * @param  {issuer} parentIssuer the `parentIssuer` is used when
+ * creating a revocable childPayment or childPurse, as in the
+ * landowner/tenant/subtenant pattern. In that pattern, the owner
+ * holds assets associated with the parent issuer, the tenant holds
+ * assets associated with the child issuer, the subtenant holds assets
+ * associated with the grandchild issuer, and so forth.
+ */
+function makePixelConfigMaker(makeUseObj, canvasSize = 10, parentIssuer) {
```

Comment:
It is a bit weird to have an optional param followed by a mandatory param. Is `parentIssuer` actually optional? If so, it should be written as `parentIssuer = undefined` or something.

Reply:
Sure, that's in